Makefile

```diff
@@ -52,18 +52,31 @@ rpm: rpm-v1 rpm-v2
 .PHONY: rpm-v1
 rpm-v1:
 	GOOS=linux GOARCH=386 make build
-	rpmbuild --define "_builddir `pwd`" --define "_version ${VERSION}" --define "buildarch noarch" -bb packaging/rpm/mkr.spec
+	rpmbuild --define "_builddir `pwd`" --define "_version ${VERSION}" --define "buildarch noarch" --target noarch -bb packaging/rpm/mkr.spec
```

Comment:
[memo] This PR adds --target option because BuildArch in spec file was removed .